packages/teams-js/test/public/media.spec.ts

```diff
@@ -46,20 +47,30 @@ describe('media', () => {
     jest.clearAllMocks();
   });
 
+  describe('isSupported', () => {
+    it('returns true if media is supported', () => {
+      Object.defineProperty(runtime.supports, 'media', { value: true });
```

Comment:
When run in VS Code the error is this => TypeError: Cannot redefine property: media.
In Jest CLI the test are passing and we have a conflict. Is it acceptable?

Reply:
this may be a node version, I ran this in VSCode to test and did not receive an error.  This was the only way I found in the docs to edit a readonly property 

Maybe we can discuss this offline to make sure I understand

Reply:
Sure I'd like to know more about it.

Reply:
so after speaking briefly, I realized that @shrshindeMSFT was saying that this creates the typeerror but the tests pass.  I was unable to create a spy or mock implementation to get around this issue, not saying that it is not possible, I just couldn't figure out how to do it.  The solution I found came from https://github.com/facebook/jest/issues/2227

Reply:
I discovered that there is a utils method designed for this purpose. Can you please not use `Object.defineProperty` and instead use `utils.setRuntimeConfig({ apiVersion: 1, supports: { media: {} } });`?

Check out calendar.spec.ts as a reference (the `isSupported` tests live at the bottom of that test file).

Reply:
sure, thanks,and that is a REALLY weird case that it only sometimes throws....impressive that you figured that out